dev/NavigationView/NavigationView.cpp

```diff
@@ -2381,24 +2545,26 @@ void NavigationView::OnNavigationViewItemOnGotFocus(const winrt::IInspectable& s
 
 void NavigationView::OnSettingsInvoked()
 {
-    auto prevItem = SelectedItem();
     auto settingsItem = m_settingsItem.get();
-    if (IsSettingsItem(prevItem))
+    if (settingsItem)
     {
-        RaiseItemInvoked(settingsItem, true /*isSettings*/);
-    }
-    else if (settingsItem)
-    {
-        SetSelectedItemAndExpectItemInvokeWhenSelectionChangedIfNotInvokedFromAPI(settingsItem);
+        OnNavigationViewItemInvoked(settingsItem);
     }
 }
 
+void NavigationView::OnPreviewKeyDown(winrt::KeyRoutedEventArgs const& e)
+{
+    m_TabKeyPrecedesFocusChange = false;
+    __super::OnPreviewKeyDown(e);
+}
+
 void NavigationView::OnKeyDown(winrt::KeyRoutedEventArgs const& e)
 {
     const auto& eventArgs = e;
     auto key = eventArgs.Key();
 
     bool handled = false;
+    m_TabKeyPrecedesFocusChange = false;
```

Comment:
Why do you set this here and in the previewkey down?

Reply:
here we can't handle arrow keys, i guess we can keep only previewkeydoun one, but something inside told me we should keep this one also.